.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -908,6 +909,9 @@ tsh bench web sessions --max=5000 --web user ls
   - [ ] Dynamodb.
   - [ ] Azure SQL Server.
   - [ ] Oracle.
+  - [ ] ClickHouse.
+- [ ] Verify auto user provisioning.
+  - [ ] Postgres.
```

Comment:
nit: should we have a separate case for `Aurora Postgres` (for the extra `rds_iam` role)?

Reply:
Yes, Your are right the slef-hosted and RDS flow is not exactly the same and it is worth to test this cases separately.